pyproject.toml

```diff
@@ -0,0 +1,156 @@
+[project]
+name        = "pact-python"
+description = "Tool for creating and verifying consumer-driven contracts using the Pact framework."
+dynamic     = ["version"]
+
+authors     = [{ name = "Matthew Balvanz", email = "matthew.balvanz@workiva.com" }]
+maintainers = [{ name = "Joshua Ellis", email = "josh@jpellis.me" }]
+
+readme = "README.md"
+license = { file = "LICENSE" }
+keywords = ["pact", "contract-testing", "testing"]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: Console",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3 :: Only",
+    "Topic :: Software Development :: Testing",
+]
+
+requires-python = ">=3.8,<4.0"
+dependencies = [
+    "click      ~= 8.1",
+    "fastapi    ~= 0.103",
```

Comment:
I noticed this and it seems equivalent to the existing requirements, however is this a dev-only dependency or does Hatch not distinguish between these during build, and figures out what is required automatically in the distribution step?

Reply:
These are actually dependencies of the library itself (not just dev dependencies). See for example:

https://github.com/pact-foundation/pact-python/blob/ed5f86c95ae0856338f0fd244c06399ce395f353/pact/cli/verify.py#L6

Upon review, some of them do appear to be dev dependencies (e.g., `fastapi` is only used within the examples). Let me double check

Reply:
Ah, it's used here: https://github.com/pact-foundation/pact-python/blob/ed5f86c95ae0856338f0fd244c06399ce395f353/pact/http_proxy.py#L4

I thought it was a different web server implementation, but this clarifies. Thanks!